moss-client/moss-client-common/src/test/java/org/xujin/moss/client/endpoint/dependency/util/PomUtilTest.java

```diff
@@ -0,0 +1,28 @@
+package org.xujin.moss.client.endpoint.dependency.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PomUtilTest {
+  @Test
```

Comment:
Looking at other test class files in this project, they appear to use 4 spaces as an indent.
It would be worth checking the indents over the whole file as these indents use 2 spaces and elsewhere there are tabs instead of spaces.